### DISCUSSION TITLE
Machine Learning System - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#210] [Announcements]
# Machine Learning System - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E7%B3%BB%E7%BB%9F/MLSys/

**Comment by ghost** [comment]
cy

**Comment by vhzy** [comment]
蹲一个，昨天刚看到准备开刷

**Reply by eatvector** [reply]
作者可以再加个ics pa

**Comment by Tim-Siu** [comment]
我这边学的是sicp-js哈哈哈 感觉挺难的哎

**Comment by xxxuwenjia** [comment]
有个油管频道     Conf42     也是相当给力了

**Comment by Delerrr** [comment]
作者您好，现在困扰我的最大的问题是没有有效的方法来寻找我所需要的课程资源。我尝试过谷歌直接搜索，但效果不理想，然后coursera、edx等平台上搜索，还是有部分课程的搜索结果为空。并且我发现您推荐的课程中有许多是有自己的网页的，并不在上述课程平台中。另外，有些课程的lecture和textbook等并没有整合到一起，而是在不同的网站上。请问您是怎么找到这些课程资源的？🙏

**Comment by ShaneHeX** [comment]
我也从 cs61a 开始的
